src/common/string_util.cc

```diff
@@ -276,6 +276,107 @@ std::pair<std::string, std::string> SplitGlob(std::string_view glob) {
   return {prefix, ""};
 }
 
+static int hexDigitToInt(const char c) {
+  if (c >= '0' && c <= '9') {
+    return c - '0';
+  } else if (c >= 'a' && c <= 'f') {
+    return c - 'a' + 10;
+  } else if (c >= 'A' && c <= 'F') {
+    return c - 'A' + 10;
+  }
```

Comment:
UNREACHABLE?

Reply:
Return 0 here to keep consistent with Redis on this.

Reply:
return 0 to keep consistent with Redis behavior.